config/settings_data.json

```diff
@@ -1 +1,31 @@
-{}
+{
+  "current": "Default",
+  "presets": {
+    "Default": {},
+    "Craft": {
+      "colors_solid_button_labels": "#EFECEC",
+      "colors_accent_1": "#2A332F",
+      "colors_accent_2": "#476154",
+      "colors_text": "#1C1A1A",
+      "colors_outline_button_labels": "#7B8382",
+      "colors_background_1": "#EFECEC",
+      "colors_background_2": "#C1BCAE",
+      "type_header_font": "americana_n4",
+      "type_body_font": "quattrocento_sans_n4",
+      "sections": {
+        "announcement-bar": {
+          "type": "announcement-bar",
+          "settings": {
+            "color_scheme": "accent-1"
+          }
+        },
```

Comment:
`accent-1` is actually already the default value for the announcement bar color scheme. So we don't need to include this here because it will default to that color anyways.

Instead, we will need to include this for the Default preset (unless the default value of the color scheme changes for the announcement bar). 

The code you have used here would work for a section that contains only one announcement bar - as in the section itself is an individual announcement bar. In this case, our announcement bar section contains blocks of type: `announcement` that each have their own settings. So we would need to provide overrides for each block instead of for the section as a whole. So it might looks something like this: 

```json
    "Default": {
      "sections": {
        "announcement-bar": {
          "type": "announcement-bar",
          "blocks": {
            "announcement-bar-0": {
              "type": "announcement",
              "settings": {
                  ... 
              }
            }
          }, 
          "block_order": [
            "announcement-bar-0" 
          ]
        }
      }
    }, 
```

Reply:
>accent-1 is actually already the default value for the announcement bar color scheme. So we don't need to include this here because it will default to that color anyways.

I had changed the default value in the section itself since it for the default style but I do it the way you mentioned instead.